script/gen_requirements_all.py

```diff
@@ -94,6 +94,8 @@
     'ring_doorbell',
     'rxv',
     'sleepyq',
+    'smhi-pkg',
+    'SoCo',
```

Comment:
Probably rebase to remove this.

Reply:
I need that right?

Reply:
We don't want to merge unrelated changes.

Reply:
Ahhh now I got ya. It is the SoCo one